php/stats.php

```diff
@@ -22,12 +23,12 @@
     $sth->execute([$trid]);
```

Comment:
I can't comment on line 17 because I haven't changed it (I hate this) but I think the existing `$uid == 1` check is wrong... I think it's written with an assumption that you can only reach this from the demo screen.